src/pages/Fox/components/FoxWifHatClaim/FoxWifHatClaim.tsx

```diff
@@ -88,20 +91,12 @@ export const FoxWifHatClaimRoutes: React.FC<FoxWifHatClaimRouteProps> = ({ accou
 
   return (
     <AnimatePresence mode='wait' initial={false}>
-      <Routes>
+      <Switch location={location.pathname}>
         <Suspense fallback={suspenseFallback}>
-          <Route
-            key={FoxWifHatClaimRoutePaths.Confirm}
-            path={FoxWifHatClaimRoutePaths.Confirm}
-            element={renderClaimConfirm()}
-          />
-          <Route
-            key={FoxWifHatClaimRoutePaths.Status}
-            path={FoxWifHatClaimRoutePaths.Status}
-            element={renderClaimStatus()}
-          />
+          <Route path={FoxWifHatClaimRoutePaths.Confirm}>{renderClaimConfirm()}</Route>
+          <Route path={FoxWifHatClaimRoutePaths.Status}>{renderClaimStatus()}</Route>
         </Suspense>
-      </Routes>
+      </Switch>
```

Comment:
FWH was broken since react-router v6 migration, fixed here, but actually agreed with @shapeshift/product to ditch it